designsafe/static/styles/main.css

```diff
@@ -1,19 +1,4 @@
-:root {
-  --ds-accent-color: #47a59d;
-  --ds-active-color: #337AB7;
-
-  /* Primary (Text, Layout) */
-  --global-color-primary--xx-light: #ffffff;
-  --global-color-primary--x-light: #f4f4f4;
-  --global-color-primary--light: #c6c6c6;
-  --global-color-primary--normal: #afafaf;
-  --global-color-primary--dark: #707070;
-  --global-color-primary--x-dark: #484848; /* ¹ */
-  --global-color-primary--xx-dark: #222222; /* ¹ */
-
-  /* Common */
-  --global-color-background--app: var(--global-color-primary--xx-light);
-}
+@import url("./variables.css");
```

Comment:
So developers can find CSS variables.[^1]

[^1]: I'd call them "settings" (ITCSS) or "(custom) properties" (actual name) but I try now to speak vocab TACC devs know.

Reply:
If the new location is not helpful, then I will move these back.